server/app/app.js

```diff
@@ -55,14 +56,43 @@ const authRoutes = require("./routes/authRoutes");
 const signpostRoute = require("./routes/signpostRoute");
 const userRoutes = require("./routes/userRoute");
 const milestoneRoute = require("./routes/milestone");
-const CodewarsRoutes = require("./routes/codewarsRoutes")
+const CodewarsRoutes = require("./routes/codewarsRoutes");
+const ReposService = require("./helpers/ReposService");
 
 app.use("/auth", authRoutes);
 app.use("/signpost", signpostRoute);
 app.use("/user", userRoutes);
 app.use("/current-milestone", milestoneRoute);
-app.use("/codewars", CodewarsRoutes )
+app.use("/codewars", CodewarsRoutes);
+
+app.get("/getAllRepos/:GithubAccount", async (req, res) => {
+  const traineeGithubAccount = req.params.GithubAccount;
 
+  try {
+    const user = await User.findOne({
+      where: { traineeGithubAccount: traineeGithubAccount },
+    });
+
+    if (!user) {
+      return res.status(404).json({ error: "User not found" });
+    }
+    const getAllRepos = await ReposService.getAllRepos(traineeGithubAccount);
+    const getAllReposwithClone = await ReposService.getAllReposWithClone(
+      getAllRepos
+    );
+    const getAllReposForCYF = await ReposService.getAllReposForCYF(
+      getAllReposwithClone
+    );
+    res.status(200).json(getAllReposForCYF);
+    const getTrainneReposNumber = await ReposService.getTrainneReposNumber(
+      getAllReposForCYF
+    );
+    // res.status(200).json(getTrainneReposNumber);
+  } catch (error) {
+    console.error(error.message);
+    res.status(500).json({ error: "Failed to fetch pull requests" });
+  }
```

Comment:
The error message is slightly misleading, as it seems that we don't fetch the number of pull requests but the number of forked repos.